### DISCUSSION TITLE
Searching for photos/videos of real usage of aaidrive, to see real-world usage

### DISCUSSION THREAD

**Discussion by Chuckame** [BimmerGestalt/AAIdrive#751] [General]
Hey there,
I'm hesitating to buy a bmw car because of all the mess when you don't have an iPhone while you still want connected services like real time traffic without the native internet option (no more 3g as announced). 

How AAIdrive is used to navigate? I've seen only one quick video showing mapbox on the HU, but how are you doing to navigate while traffic is changing?

**Comment by hufman** [comment]
AAIdrive simply asks Mapbox for a route, which presumably factors in traffic conditions, and then adds that as a polyline to the displayed map. It doesn't do any turn-by-turn directions or automatic rerouting.

**Comment by im0-ru** [comment]
Modern BMWs (after 2020 for sure) fully support Android Auto. You will not need to use AAIdrive.
